universal-application-tool-0.0.1/tailwind.config.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
       colors: {
         orange: colors.orange,
         teal: colors.teal,
+        beige: {
```

Comment:
I was avoiding doing this because I didn't want to stray from the Tailwind color palettes but I guess here we are.

For the initial release with the city, I don't think they'll want a beige background. Should we standardize on something else based on their style guide?

Reply:
Great question. I'll ask in the #product-ux slack channel whether we've gotten feedback on the beige in the mocks from the City.

Reply:
Great question - asked PM about this in slack.

Reply:
We'll probably be using config/CSS files like the ones here: https://play.tailwindcss.com/JzARWTvFQP?file=css (same link I sent in slack)